src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -565,6 +567,36 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return New LazyIsTypeExpression(operand, isType, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundLateInvocationOperation(boundLateInvocation As BoundLateInvocation) As IOperation
+            Dim expression As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundLateInvocation.Member))
+
+            Dim isDynamicPropertyReference = boundLateInvocation.MethodOrPropertyGroupOpt?.Kind = BoundKind.PropertyGroup
```

Comment:
>isDynamicPropertyReference [](start = 16, length = 26)

I believe It is not always possible to accurately calculate this value for VB.

Reply:
@AlekseyTs do you have an example where this could likely be wrong? I was able to write unit tests where this value is correctly computed to true for property reference and false for method reference.

Reply:
> do you have an example where this could likely be wrong?

Aren't there cases when we simply don't know whether the name refers to a property, or to a method?

Reply:
I get your point, lets discuss the correct way to represent this in the next design meeting.

Reply:
I believe that from VB point of view there is no (o very little) difference between late bound method invocation or late bound property indexing. At the same time, it looks like there is no such thing as dynamic property indexing on C# side.